src/frequenz/sdk/microgrid/_data_pipeline.py

```diff
@@ -433,6 +471,28 @@ def logical_meter() -> LogicalMeter:
     return _get().logical_meter()
 
 
+def consumer() -> Consumer:
+    """Return the consumer instance.
+
+    If a Consumer instance doesn't exist, a new one is created and returned.
```

Comment:
Also, I guess this is how things are documented for now in this module, but I would omit this, this is completely irrelevant to the user.